samples/snippets/feature_store_service/batch_create_features_sample.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START aiplatform_batch_create_features_sample]
+from google.cloud import aiplatform_v1beta1 as aiplatform
+
+
+def batch_create_features_sample(
+    project: str,
+    featurestore_id: str,
+    entity_type_id: str,
+    location: str = "us-central1",
+    api_endpoint: str = "us-central1-aiplatform.googleapis.com",
+    timeout: int = 300,
+):
+    # The AI Platform services require regional API endpoints.
+    client_options = {"api_endpoint": api_endpoint}
+    # Initialize client that will be used to create and send requests.
+    # This client only needs to be created once, and can be reused for multiple requests.
+    client = aiplatform.FeaturestoreServiceClient(client_options=client_options)
+    parent = f"projects/{project}/locations/{location}/featurestores/{featurestore_id}/entityTypes/{entity_type_id}"
+    requests = [
+        aiplatform.CreateFeatureRequest(
+            feature=aiplatform.Feature(
+                value_type=aiplatform.Feature.ValueType.INT64, description="User age",
+            ),
+            feature_id="age",
+        ),
+        aiplatform.CreateFeatureRequest(
+            feature=aiplatform.Feature(
+                value_type=aiplatform.Feature.ValueType.STRING,
```

Comment:
Should these be part of input argument, to be consistent with all the other sample snippet. Same for ID, description

Reply:
No, sample snippets are for user referencing purposes:
https://github.com/googleapis/python-aiplatform/pull/713#discussion_r710445400

It would be unclear how these requests got constructed if we pass them from the test file.

Reply:
I mean it's not clear for me to see why feature_id and entity_id is in the args, but not the others. If to provide sample for user to reference, then they can all be hard coded inside this function (note you also do that for entity type in the ingestion sample).